jp2_online/templates/tosp_auth/login_form.html

```diff
@@ -0,0 +1,28 @@
+<div class="row text-center">
```

Comment:
Do you mean like this?

http://vinceg.github.io/Bootstrap-Admin-Theme-3/login.html